### PR TITLE
Release 6.4.0

### PR DESCRIPTION
**Pull request by djbe**
_This release PR targets the `stable` branch. To just review the changes since the release PR was cut from `develop` branch, see [here](https://github.com/SwiftGen/SwiftGen/compare/develop...release/6.4.0)_

### CODE REVIEW

**Comment by SwiftGen-Eve** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should make sure that <code>Podfile.lock</code> and <code>Package.resolved</code> are changed in sync</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This is a Release PR</td>
    </tr>
  </tbody>
</table>

Hey 👋 I'm Eve, the friendly bot watching over SwiftGen 🤖

Thanks a lot for your contribution!



---

### ChangeLog check

✅  All entries seems OK (end with period + 2 spaces, correct links)


### Release version check

| Check                                         | Status       |
| --------------------------------------------- | ------------ |
| Bundler installed                             | ✅           |
| SwiftGen.podspec                              | 👉  6.4.0    |
| SwiftGenKit.podspec                           | 👉  6.4.0    |
| SwiftGen & SwiftGenKit versions equal         | ✅           |
| SwiftGen-Info.plist version matches           | ✅           |
| SwiftGenKit-Info.plist version matches        | ✅           |
| StencilSwiftKit up-to-date (latest: 2.7.2)    | ✅           |
| CHANGELOG: Release entry added                | ✅           |
| CHANGELOG: No develop entry                   | ✅           |
| README: pod version ~> 6.0                    | ✅           |


Once you fix those tiny nitpickings above, we should be good to go! 🙌

ℹ️ _I will update this comment as you add new commits_

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by AliSoftware** [comment]
~Looking at the CHANGELOG since develop it seems to me that this release will only contain bug fixes and internal changes, right? So shouldn't this be a 6.3.1?~

[Edit] nvm I'm tired tonight. Changes since stable contains way more things obviously, including new features, so version number seems ok

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,51 @@
...
+## 6.4.0
+
+### New Features
+
+* The built-in templates will now correctly handle Swift Package Manager resources, using `Bundle.module` if it's available. As before, you can override the used `Bundle` using the `bundle` or `lookupFunction` template parameters.  
+  [Arnaud Dorgans](https://github.com/arnauddorgans)
+  [#763](https://github.com/SwiftGen/SwiftGen/pull/763)
+* Added `config generate-xcfilelist` subcommand to generate input and output `xcfilelist`s based on your configuration file. These files should then be used in an Xcode build step that executes `swiftgen config run`. Don't forget to regenerate the file lists after adding/removing resources in your project in a way that might impact them.  
+  [@CraigSiemens](https://github.com/CraigSiemens)
+  [#441](https://github.com/SwiftGen/SwiftGen/issues/441)
```
Comment (AliSoftware) [inline CHANGELOG.md:14]: 😱 I missed that PR, that's an awesome new feature! 🎉

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,51 @@
...
+  [David Jennes](https://github.com/djbe)
+  [#778](https://github.com/SwiftGen/SwiftGen/pull/778)
+
+### Bug Fixes
+
+* Plist: Update the parsing strategy (using `Codable`) to fix parsing of `Bool` values as `Integer` in some cases.  
+  [@fortmarek](https://github.com/fortmarek)
+  [Olivier Halligon](https://github.com/AliSoftware)
+  [#779](https://github.com/SwiftGen/SwiftGen/pull/779)
+* JSON/Plist/YAML: the `inline` templates incorrectly generated `1`/`0` as values, instead of `true`/`false` as expected.
```
Comment (AliSoftware) [inline CHANGELOG.md]: Forgot to nitpick on this during the PR, but feels a bit strange to have past tense ("this thing was not working") without mention of present tense (eg phrasing like "and is now fixed")

What do you think about wording it more like "Fixed an issue with inline templates which incorrectly generated […]"?

**Review by AliSoftware — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (djbe) [reply CHANGELOG.md:14]: Yeah big props to @CraigSiemens for enduring through the review process 😉 😅

Comment (CraigSiemens) [reply CHANGELOG.md:14]: 😊 I think the PR went fairly smoothly. I've definitely had worse.

**Comment by djbe** [comment]
```
--------------------------------------------------------------------------------
🎉   Congrats

🚀   SwiftGen (6.4.0) successfully published
📅   October 6th, 00:00
🌎   https://cocoapods.org/pods/SwiftGen
👍   Tell your friends!
--------------------------------------------------------------------------------
```

Now also available on Homebrew!

**Pull request merged**